ktor-http/common/src/io/ktor/http/HttpUrlEncoded.kt

```diff
@@ -11,7 +11,11 @@ import io.ktor.utils.io.charsets.*
  *
  * [Report a problem](https://ktor.io/feedback/?fqname=io.ktor.http.parseUrlEncodedParameters)
  */
-public fun String.parseUrlEncodedParameters(defaultEncoding: Charset = Charsets.UTF_8, limit: Int = 1000): Parameters {
+public fun String.parseUrlEncodedParameters(
+    defaultEncoding: Charset = Charsets.UTF_8,
+    limit: Int = 1000,
+    plusIsSpace: Boolean = false
+): Parameters {
```

Comment:
Could you also preserve a function with current signature to ensure this change is binary compatible?
Something like this:

```kotlin
@Deprecated("Maintained for binary compatibility", DeprecationLevel.HIDDEN)
public fun String.parseUrlEncodedParameters(
    defaultEncoding: Charset = Charsets.UTF_8,
    limit: Int = 1000,
): Parameters = parseUrlEncodedParameters(defaultEncoding, limit, plusIsSpace = false)
```